features/admin.ai.v1/components/branding-ai-banner.tsx

```diff
@@ -0,0 +1,200 @@
+                                            >
+                                                <GenericIcon
+                                                    className="branding-card-banner-input-button-icon"
+                                                    icon={ AIIcon }
+                                                />
+                                                { t("branding:ai.banner.input.button") }
+                                            </Button>
+                                        </div>
+                                    </div>
+                                </div>
+                            </Segment>
+                        ) }
+                        { bannerState === BannerState.COLLAPSED && (
+                            <Segment>
+                                <div className="branding-card-banner-collapsed">
+                                    <div>
+                                        <Header as="h3" className="branding-card-banner-collapsed-heading">
+                                            { t("branding:ai.banner.input.heading") }</Header>
+                                        <p>{ t("branding:ai.banner.collapsed.subHeading") }
+                                            <DocumentationLink
+                                                link={ "develop.applications.editApplication.asgardeoTryitApplication" +
+                                    ".general.learnMore" }
+                                                isLinkRef
+                                            >
+                                                <Trans
+                                                    i18nKey={ "extensions:common.learnMore" }
+                                                >
+                                        Learn More
+                                                </Trans>
+                                            </DocumentationLink>
+                                        </p>
+                                    </div>
+                                    <Button
+                                        onClick={ () => setBannerState(BannerState.INPUT) }
+                                        color="secondary"
+                                        variant="outlined"
+                                    >
+                                        <GenericIcon
+                                            className="branding-card-banner-collapsed-button"
+                                            icon={ AIIcon }
+                                        />
+                                        { t("branding:ai.banner.collapsed.button") }
+                                    </Button>
+                                </div>
+                            </Segment>
+                        ) }
+                    </div>)
+            }
+        </>
+    );};
+
+BrandingAIBanner.defaultProps = {
+
+};
```

Comment:
Let's remove this if we not passing any default props.

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/5961/commits/891f646b2c63094d4467060253aa82c4bd6751b5